signer/src/lib.rs

```diff
@@ -125,6 +125,19 @@ pub const WITHDRAWAL_MIN_CONFIRMATIONS: u64 = 6;
 /// https://github.com/stacks-network/sbtc/issues/620.
 pub const WITHDRAWAL_BLOCKS_EXPIRY: u64 = 24;
 
+/// This is the number of bitcoin blocks prior to [`WITHDRAWAL_BLOCKS_EXPIRY`]
+/// in which the coordinator will cease to include a withdrawal request in sweep
+/// proposals.
+///
+/// This is the value in https://github.com/stacks-network/sbtc/issues/1363's
+/// proposed "buffer for expiring requests" section.
+pub const WITHDRAWAL_EXPIRY_BUFFER: u64 = 6;
+
+/// This is the number of bitcoin blocks that a withdrawal request will remain
+/// live before it is considered "soft expired". This is the point at which the
+/// coordinator will cease to include a withdrawal request in sweep proposals.
+pub const WITHDRAWAL_BLOCKS_SOFT_EXPIRY: u64 = WITHDRAWAL_BLOCKS_EXPIRY - WITHDRAWAL_EXPIRY_BUFFER;
```

Comment:
It seems we are only using this in a test (`assert_processes_withdrawals`), should we ditch the const and compute the difference there?

Reply:
Yeah I was actually thinking the same; added it on daniel's request, but we assumed it would be used more widely. I'll remove it for now.

Reply:
Removed it in [b88da7c](https://github.com/stacks-network/sbtc/pull/1399/commits/b88da7c88b71db22f3cde725a505c4cc564d5d55)